query/postgres/postgres_db_server_geo_redundat_backup_enabled.sql

```diff
@@ -0,0 +1,19 @@
+select
```

Comment:
Please rename file to `query/postgres/postgres_db_server_geo_redundant_backup_enabled.sql`

Reply:
Updated as suggested.